staging/src/k8s.io/apiserver/pkg/server/options/authorization.go

```diff
@@ -63,9 +63,19 @@ func NewDelegatingAuthorizationOptions() *DelegatingAuthorizationOptions {
 		// very low for responsiveness, but high enough to handle storms
 		AllowCacheTTL: 10 * time.Second,
 		DenyCacheTTL:  10 * time.Second,
+
+		// by default, allow health checks to anyone (but no details).  This can overridden by clearing the field.
+		// this does not match /healthz/ or any subpath like /healthz/foo
+		AlwaysAllowPaths: []string{"/healthz"},
```

Comment:
This was also in the original PR, but we decided against setting a default. /cc @liggitt

Reply:
> This was also in the original PR, but we decided against setting a default. /cc @liggitt

Pared this back to providing a `With` method, but I do still think that setting a default makes a lot more sense.